crates/util/src/paths.rs

```diff
@@ -44,6 +44,7 @@ lazy_static::lazy_static! {
     pub static ref LOG: PathBuf = LOGS_DIR.join("Zed.log");
     pub static ref OLD_LOG: PathBuf = LOGS_DIR.join("Zed.log.old");
     pub static ref LOCAL_SETTINGS_RELATIVE_PATH: &'static Path = Path::new(".zed/settings.json");
+    pub static ref TEMP_DIR: PathBuf = CONFIG_DIR.join("tmp");
```

Comment:
Small nit, but the `XDG_CACHE_DIR` is a better candidate for this.

https://wiki.archlinux.org/title/XDG_Base_Directory

Reply:
Other alternative could be to write the file to the same directory as the target path, it would ensure that it's on the same filesystem.

Reply:
Good point! write to the same directory would be a better choice